packages/server/src/scim/utils.ts

```diff
@@ -244,3 +299,49 @@ export function convertToScimListResponse<T>(Resources: T[]): ScimListResponse<T
     Resources,
   };
 }
+
+/**
+ * Converts a SCIM patch request to a JSONPatch request.
+ * SCIM PATCH operations are not the same as JSONPatch operations.
+ * SCIM PATCH can omit "path" if the operation is "add" or "replace".
+ * SCIM PATCH "path" values can omit the leading "/".
+ *
+ * See: https://www.rfc-editor.org/rfc/rfc7644#section-3.5.2
+ *
+ * @param scimPatch - The original SCIM patch request.
+ * @returns The converted JSONPatch request.
+ */
+export function convertScimToJsonPatch(scimPatch: ScimPatchRequest): Operation[] {
+  return scimPatch.Operations.flatMap((inputOperation) => {
+    const { op, path, value } = inputOperation;
+
+    if (op !== 'add' && op !== 'remove' && op !== 'replace' && op !== 'copy' && op !== 'move' && op !== 'test') {
```

Comment:
Aren't the only allowed operations "add", "remove", or "replace"?

Reply:
Hm, good call.  I thought it supported all.  I'll double check the SCIM spec.